### PR TITLE
Blacklist command CLI handler

### PR DESCRIPTION
**Pull request by tarun-29**
This PR adds logic to check for black list command and throw error

### CODE REVIEW

File: internal/constants/constants.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package constants
+
+var BlacklistedCommands = []string{
```
Comment (lucifercr07) [inline internal/constants/constants.go:3]: Let's rename this to `BlockedCommandsList`

File: internal/server/httpServer.go [hunk 2]
```diff
@@ -87,10 +87,25 @@ func (s *HTTPServer) CliHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Check for blacklisted commands
+	if containsBlacklistedCommand(strings.ToUpper(diceCmds.Cmd)) {
+		http.Error(w, "Error: One or more commands are not allowed", http.StatusForbidden)
```
Comment (lucifercr07) [inline internal/server/httpServer.go:92]: API can't execute more than 1 command at once.

File: internal/server/httpServer.go [hunk 3]
```diff
@@ -87,10 +87,25 @@ func (s *HTTPServer) CliHandler(w http.ResponseWriter, r *http.Request) {
...
+
 	resp := s.DiceClient.ExecuteCommand(diceCmds)
 	util.JSONResponse(w, http.StatusOK, resp)
 }
 
 func (s *HTTPServer) SearchHandler(w http.ResponseWriter, request *http.Request) {
 	util.JSONResponse(w, http.StatusOK, map[string]string{"message": "Search results"})
 }
+
+func containsBlacklistedCommand(Cmds string) bool {
```
Comment (lucifercr07) [inline internal/server/httpServer.go:104]: Let's rename the method also.

**Review by lucifercr07 — CHANGES_REQUESTED** [review]

**Review by lucifercr07 — CHANGES_REQUESTED** [review]
Please add tests around this.

File: internal/constants/constants.go (hunk 1 above)
Comment (tarun-29) [reply internal/constants/constants.go:3]: resolved

File: internal/server/httpServer.go (hunk 2 above)
Comment (tarun-29) [reply internal/server/httpServer.go:92]: updated the message

File: internal/server/httpServer.go (hunk 3 above)
Comment (tarun-29) [reply internal/server/httpServer.go:104]: done

File: internal/constants/constants.go (hunk 1 above)
Comment (lucifercr07) [reply internal/constants/constants.go:3]: It's `BlockedCommandsList` please fix it.

**Comment by tarun-29** [comment]
> Please add tests around this.

hi @lucifercr07 shall i add it for whole CliHandler or only around function which isBlockedCommand?

**Comment by lucifercr07** [comment]
Since [issue](https://github.com/DiceDB/dice/issues/897) was assigned to another user and their PR is already created, it won't be good if we continue with this PR. Closing this PR, really thanks for your contribution.

**Pull request closed without merging**